src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Financial/RoatpFinancialApplicationViewModel.cs

```diff
@@ -40,25 +39,29 @@ public RoatpFinancialApplicationViewModel(Guid id, Guid applicationId, List<Sect
                 ApplicationId = applicationId;
             }
 
-            SetupGradeAndFinancialDueDate(grade);
             OrgId = application.OrganisationId;
+            FinancialReviewDetails = application.ApplyData.FinancialReviewDetails;
+            SetupGradeAndFinancialDueDate();
+
+            LegalName = application.ApplyData.ApplyDetails.OrganisationName;
+            TradingName = application.ApplyData.ApplyDetails.TradingName;
+            Ukprn = Convert.ToInt32(application.ApplyData.ApplyDetails.UKPRN);
+            ProviderName = application.ApplyData.ApplyDetails.OrganisationName;
         }
 
-        private void SetupGradeAndFinancialDueDate(FinancialGrade grade)
+        private void SetupGradeAndFinancialDueDate()
```

Comment:
This looks confusing in that you're setting up stuff but not passing in a DTO / class with the information inside

Reply:
Done